lmfit/model.py

```diff
@@ -1885,13 +1885,14 @@ def loads(self, s, funcdefs=None, **kws):
         self.model = _buildmodel(decode4js(modres['model']), funcdefs=funcdefs)
 
         # params
+        funcdefs.pop(self.model.func.__name__)  # Remove model function to not pass into _asteval
```

Comment:
This may not be the most elegant means to do this, but something I noticed was that if the model function is not removed from the funcdefs argument, it gets added to the `_asteval.symtable`. Although a small change, undesired changes to the objects through de/serialization should be minimized, so I think it should be handled somehow.